### DISCUSSION TITLE
how to hide the screen colorpicker icon of ColorPicker

### DISCUSSION THREAD

**Discussion by WillemWe** [HandyOrg/HandyControl#995] [Q&A]
![image](https://user-images.githubusercontent.com/49357323/134485771-f00dcc9f-e6a7-4916-829d-c01f605bcf97.png)
I want to hide the screen colorpicker icon of ColorPicker,but I don't how to do ,I didn't see any property to set it,can you help me?thanks

**Comment by ghost1372** [comment]
You cant hide it

**Reply by WillemWe** [reply]
Is there any other method to solve it? I used ColorPicker in a Popup,when the Popup is show,and I clicked the icon ,when the Popup hide,mouse icon were changed and can't back,how can iI do it?

**Reply by ghost1372** [reply]
This seems to be a bug. Can you send a gif?

**Reply by WillemWe** [reply]
> This seems to be a bug. Can you send a gif?

![test](https://user-images.githubusercontent.com/49357323/134494246-e0c53999-c70a-4ff6-83db-441daa1f76ad.gif)

**Comment by Yopler** [comment]
The colorpicker icon is inside an ToogleButton using the `ToggleButtonCustom` Style. See [**ColorPickerBaseStyle.xaml** - _line 185_](https://github.com/HandyOrg/HandyControl/blob/master/src/Shared/HandyControl_Shared/Themes/Styles/Base/ColorPickerBaseStyle.xaml#L185)
You can try to overwrite this style before using the ColorPicker. You can use the property Visibility to hide something. This is njot really clean but it can be a way 😅
Example :
```xaml
<Style x:Key="ToggleButtonCustom" TargetType="ToogleButton">
     <Setter Property="Visibility" Value="Collapsed"/>
</Style>
```
